.github/pull-request-template.md

```diff
@@ -1,28 +1,31 @@
 <!-- General PR guidelines:
 
-Most PRs should be opened against the main branch.
+Most PRs should be opened against the main branch in the
+[Docs GitHub repository](https://github.com/knative/docs).
```

Comment:
This is referred to as the 

> `knative/docs` repo

inside `structure.md` (also touched in this PR).

Shall we decide on using either "`knative/docs` GitHub repository" _or_ "Docs GitHub repository" consistently, since both changes are in the same PR?

If you think the hyperlink is explicit enough that's perfectly fine with me! In this case feel free to ignore.

Reply:
Thanks, I'll apply that change.